.eslintrc.yml

```diff
@@ -9,3 +9,11 @@ settings:
     node:
       paths: ['./src']
       extensions: ['.js', '.jsx']
+
+overrides:
```

Comment:
Maybe you could move this section to @oacore/configs and release it. And after that finish this PR.